manager/misc.go

```diff
@@ -184,7 +185,6 @@ func (m *VolumeManager) generateSupportBundle(bundleDir string, bundleMeta *Bund
 
 	logsDir := filepath.Join(bundleDir, "logs")
 	m.generateSupportBundleLogs(logsDir, errLog, sb)
-	sb.ProgressPercentage = BundleProgressPercentageTotal
```

Comment:
Should add
`sb.ProgressPercentage = BundleProgressPercentageYaml + BundleProgressPercentageLogs` here. To keep it consistent with the yaml generation.

Reply:
BundleProgressPercentageYaml + BundleProgressPercentageLogs will add up to 95%. I summed it up to 100% as we have 5% compression time.

Reply:
I meant in order to keep consistency in the code, we should set the percentage here as well. So later when people reading the code, they would know it's the second step and what percentage to expect after this step is done.